doc/source/tutorials/quickstart.rst

```diff
@@ -146,23 +146,24 @@ But in order to give you an example that you can start with from scratch, that's
 Step 1: Make a vessel
 ^^^^^^^^^^^^^^^^^^^^^
 
-Make any sort of rocket that can lift you to orbit that fills the following pattern:
+Please make the vessel you see here:
```

Comment:
I'd be a bit more explicit here and add something to the effect "This tutorial doesn't work well for all rocket designs, and is tailored to this specific craft.  You need to build this exact craft to work with this example."  Rather than "please" make the vessel you see here.  Imply that it won't work at all if you make a different one (not entirely true, but it gets the point across).